pwndbg/gdblib/disasm/arm.py

```diff
@@ -57,6 +57,17 @@ def _condition(self, instruction: PwndbgInstruction, emu: Emulator) -> Instructi
 
         return InstructionCondition.TRUE if bool(cc) else InstructionCondition.FALSE
 
+    @override
+    def _resolve_target(self, instruction: PwndbgInstruction, emu: Emulator | None, call=False):
+        target = super()._resolve_target(instruction, emu, call)
+        if target is not None:
+            # On interworking branches - branches that can enable Thumb mode - the target of a jump
+            # has the least significant bit set to 1. This is not actually written to the PC
+            # and instead the CPU puts it into the Thumb mode register bit.
+            # This means we can clear the least significant bit of the target.
```

Comment:
```suggestion
            # This means we can clear the least significant bit of the target.
```
Technically, I believe we have to do it.

Reply:
I updated the comment to indicate this.